### PR TITLE
Project skeleton

### PR DESCRIPTION
**Pull request by AckonSamuel**
In this milestone of Module 2, below are the pieces of stuff implemented:

- Gitflow is implemented
- Project built with webpack
- Plain HTML markup with minimum styling used to make the layout
- ES6 modules are used

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,42 @@
-# curriculum-databases-projects-template
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Leaderboard
```
Comment (julie-ify) [inline README.md:2]: - [x] Kindly add a detailed description of your project where users get to know about your project features and more.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @AckonSamuel,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

- [x] **[REQUIRED]** Kindly set your `dev` branch to the [default branch](https://docs.github.com/en/github/administering-a-repository/managing-branches-in-your-repository/changing-the-default-branch#changing-the-default-branch) so that you can automatically be asked to compare your feature branch with the development branch once you create pull requests.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AckonSamuel** [comment]
Thanks for the review

File: README.md (hunk 1 above)
Comment (AckonSamuel) [reply README.md:2]: Done!

**Review by rustam-tolipov — APPROVED** [review]
## Hello @AckonSamuel 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**